db/migrate/20201216184644_add_import_count_and_failed_count_and_userid_to_imported_items.rb

```diff
@@ -1,7 +1,8 @@
 class AddImportCountAndFailedCountAndUseridToImportedItems < ActiveRecord::Migration[6.0]
   def change
     add_column :imported_items, :imported, :integer
-    add_column :imported_items, :failed, :integer
-    add_column :imported_items, :uploaded_by_user_id, :bigint
+    add_column :imported_items, :rejected, :integer
   end
+
+  add_reference :imported_items, :user, foreign_key: true, index: { algorithm: :concurrently }
```

Comment:
this probably just my ignorance, buy why does the `add_reference` go outside of the `def change`?

Reply:
Its to do with strong migrations, well at least I ran into issues with it telling me what to do.